README.rst

```diff
@@ -115,11 +142,32 @@ sends the character to the ``window manager`` (DWM, metacity, i3, etc), so the
 The graphical API of the window  that receives the character prints the
 appropiate font symbol in the appropiate focused field.
 
+
+Parse URL
+---------
+
+* The browser has now the following information contained in the URL (Uniform
+  Resource Locator):
+
+    - ``Protocol``  "http"
+        Use 'Hyper Text Transfer Protocol'
+
+    - ``Domain name``  "www.google.com"
+        Connect to host 'www.google.com'
+
+    - ``Web service type``  ".com"
```

Comment:
I would remove these two lines... the information is contained in 155-156, and .com is just a TLD. While it's theoretically supposed to be for 'commercial' services, that's not really respected these days, any more than .ly is only for websites hosted in Lybia.

You could add a note to the 'domain name' section saying that .com stands for Commercial.